AnimateSvg.js

```diff
@@ -15,20 +14,71 @@ export default function AnimateSvg(props) {
   });
   const sizeAnim = useRef(new Animated.Value(1)).current;
 
-  const { toggle } = useIsShake();
+  const { isShakeTriggered, setIsToggleReady } = props.useIsShake;
+  const { hasToggledBefore } = props.toggledBefore;
 
   useEffect(() => {
     Animated.timing(fadeAnim, {
       toValue: 1,
       duration: SVG_LOAD_DURATION,
       useNativeDriver: true,
-    }).start();
+    }).start(({ finished }) => {
+      setIsToggleReady(true);
+      console.log("isToggleReady set true");
+    });
   }, []);
 
-  // LogoSvg fades out first time
-  if (toggle && !hasToggledBefore && !isRegistered) {
-    setIsRegistered(true);
-    setHasToggledBefore(true);
+  function logoAnimationOutInitialization() {
```

Comment:
Animation compositions moved from conditional logic into function definitions to make code more legible and descriptive of what the animation is doing.

Reply:
remove this, clear from code change